tests/functional/sheet.js

```diff
@@ -590,6 +605,12 @@ describe('Functional: sheet', () => {
       expect(getCss(getStyle())).to.be(removeWhitespace(sheet.toString()))
     })
 
+    it('should render sheet with updated props after attach', () => {
+      sheet.attach().update({color: 'green'})
+      const style = removeVendorPrefixes(getCssFromSheet(sheet))
```

Comment:
because some browsers automatically add their vendors to the keyframes rule, lol